building-epics/getting-epics7-on-rtems5-beatnik.rst

```diff
@@ -0,0 +1,4365 @@
+   ************ MBUF STATISTICS ************
+   mbufs:16384    clusters:2560    free:2520
+   drops:   0       waits:   0  drains:   0
+         free:16343         data:41          header:0           socket:0
+          pcb:0           rtable:0           htable:0           atable:0
+       soname:0           soopts:0           ftable:0           rights:0
+       ifaddr:0          control:0          oobdata:0
+
+   Destination     Gateway/Mask/Hw    Flags     Refs     Use Expire Interface
+   default         141.14.128.128     UGS         0        0      0 mve1
+   127.0.0.1       127.0.0.1          UH          0        0      0 lo0
+   141.14.128.0    255.255.240.0      U           0        0      7 mve1
+   141.14.128.128  00:24:38:93:C8:00  UHL         1        0   1209 mve1
+   ************ IP Statistics ************
+                total packets received          39
+     packets rcvd for unreachable dest           1
+    datagrams delivered to upper level          38
+       total ip packets generated here           3
+
+   ************ ICMP Statistics ************
+
+   ************ UDP Statistics ************
+                   total input packets          40
+                     no socket on port           4
+        of above, arrived as broadcast          42
+                  total output packets           3
+
+   ************ TCP Statistics ************
+
+
+   ***** Setting up file system *****
+   ***** Using compiled in file data *****
+   -> /iocshTestBadArgIndirect.cmd - ok
+   -> /iocshTestBadArg.cmd - ok
+   -> /iocshTestSuccessIndirect.cmd - ok
+   -> /iocshTestSuccess.cmd - ok
+   ***** Initializing NFS *****
+    check for time registered , C++ initialization ...
+    Will try to start telnetd with prio 99 ...
+   syslog: telnetd: started successfully on port 23
+    telnetd initialized with result 0
+   ***** Preparing EPICS application *****
+   chdir("/")
+   ***** Starting EPICS application *****
+   Backwards time errors prevented 0 times.
+
+   Current Time Providers:
+       "OS Clock", priority = 999
+           Current Time is 2021-01-07 16:36:08.056273.
+
+   Event Time Providers:
+           No Providers registered.
+
+   ***** epicsThreadTest *****
+   1..15
+   # System has 1 CPUs
+   ok  1 - ncpus > 0
+   # main() thread 0x2d47e0
+   ok  2 - Join delayed parent (2.00998 seconds)
+   ok  3 - Join tests #1 completed
+   ok  4 - Join delayed parent (2.00998 seconds)
+   ok  5 - Join tests #2 completed
+   ok  6 - pget == pset
+   ok  7 - thread.getPriority() == epicsThreadGetPriority(self)
+   ok  8 - pget == pset
+   ok  9 - pget == pset
+   ok 10 - thread.getPriority() == epicsThreadGetPriority(self)
+   ok 11 - thread.getPriority() == epicsThreadGetPriority(self)
+   ok 12 - threadA epicsThreadIsOkToBlock() = 0
+   ok 13 - threadB epicsThreadIsOkToBlock() = 1
+   ok 14 - infoB.didSomething
+   ok 15 - infoA.didSomething
+
+       Results
+       =======
+          Tests: 15
+         Passed:  15 = 100.00%
+
```

Comment:
I do not see the value of adding the complete EPICS test output here, with not tests failing.
Any new version of EPICS Base is likely to produce slightly different output, anyway.

What about an ellipsis and cutting to the last handful of lines?

Reply:
The idea was to show that really all tests were successful and nothing remained open.